automation/system_test/cleanup.py

```diff
@@ -81,6 +88,26 @@ async def _collect_image_tags(
     return tags
 
 
+def _remove_image_from_datanode_docker():
+    """Remove image from datanode docker"""
+    subprocess.run(
+        [
+            "docker",
+            "images",
+            "--format",
+            "'{{.Repository }}:{{.Tag}}'",
+            "|",
+            "grep",
+            "mlrun",
+            "|",
+            "xargs",
+            "docker",
```

Comment:
add `--no-run-if-empty` flag to xargs so if grep yields no images, then xargs wont run